.github/actions/spell-check/patterns/patterns.txt

```diff
@@ -1,3 +1,4 @@
+https://(?:(?:www\.|)microsoft\.com)/[-a-zA-Z0-9?&=]*
```

Comment:
You're missing `_`

Reply:
Yeah ... wait what? I'm sorry, can you elaborate?

Reply:
`-` (dash) != `_` (underscore). The URL uses both 😯